src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -1,12 +1,31 @@
 {% trans_default_domain "locationview" %}
 {{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_location_actions', {'contentId': content.id, 'redirectLocationId': location.id})}) }}
 
+<dialog id="confirm-location-removal-{{ location.id }}" class="ez-modal">
+    <section>
+        <p>{{ 'confirm.remove.location'|trans()|desc('Are you sure you want to remove selected locations?') }}</p>
+    </section>
+    <footer>
+        <button type="button" class="ez-button ez-button-secondary ez-js-close-modal">
+            {{ 'confirmbox.cancel'|trans({}, 'confirmbox')|desc('Cancel') }}
+        </button>
+        {{
+            form_widget(actionsForm.delete, {
+                'label': 'confirmbox.confirm'|trans({}, 'confirmbox')|desc('Confirm'),
+                'attr': {'class': 'ez-button ez-button-primary ez-js-close-modal'}
```

Comment:
If the text of the button is now `Delete`, then the style of the button should change from `.ez-button-primary` to '.ez-button-negative`